### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AnaisUrlichs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,15 @@ By using Gitpod in this project, you agree to opt-in to basic, anonymous analyti
   [yarn]: https://yarnpkg.com/
   [AssemblyScript]: https://assemblyscript.org/
   [Wasm]: https://webassembly.org/
+  
+
+# Course on FTs on NEAR
+
+If you want to learn more about fungible tokens and create your own project, head over to the course on ["Building and Issuing Fungible Tokens on NEAR"](https://vitalpoint.ai/academy/building-and-issuing-fungible-tokens-on-near/). The course was created by one of our Guilds, [Vital Point AI](https://github.com/ALuhning).
```
Comment (chadoh) [inline README.md:49]: A small consideration: who or what is "our"? What is the entity that owns the Guilds? If I write "our" or "we" in READMEs like this, with what authority's voice am I speaking? The NEAR Foundation? Near, Inc? Those seem like entities that could reasonably use a possessive pronoun. I'm not sure that the NEAR Collective, decentralized as it strives to be, can reasonably use possessive pronouns. Who gave you, or this repo, permission to speak on behalf of the whole cacophonous collective? Who could?

I've been avoiding "we" and possessives in this sort of writing, and instead saying things like "This course was created by a NEAR Guild"

Comment (AnaisUrlichs) [reply README.md:49]: Uff, you are right I should not write it that way. It is it's own entity -- let me change it

**Comment by vgrichina** [comment]
@AnaisUrlichs @chadoh is this waiting for https://vitalpoint.ai/academy/building-and-issuing-fungible-tokens-on-near/?

**Comment by AnaisUrlichs** [comment]
@vgrichina this should be ready to merge given that @chadoh is cool with the changes

**Comment by vgrichina** [comment]
@AnaisUrlichs I'm going to merge, but https://vitalpoint.ai/academy/building-and-issuing-fungible-tokens-on-near/ seems unresponsive

**Comment by chadoh** [comment]
@AnaisUrlichs I think you could have merged 😊 

let me know if you don't have the proper access and I can fix that for you!

**Comment by AnaisUrlichs** [comment]
Sorry, will merge next time :grin: @chadoh

**Pull request merged**